homeassistant/components/remote/__init__.py

```diff
@@ -111,24 +113,26 @@ class RemoteDevice(ToggleEntity):
     """Representation of a remote."""
 
     @property
-    def supported_features(self):
+    def supported_features(self) -> int:
         """Flag supported features."""
         return 0
 
-    def send_command(self, command, **kwargs):
-        """Send a command to a device."""
+    def send_command(self, command: Iterable[str], **kwargs: Any) -> None:
+        """Send commands to a device."""
         raise NotImplementedError()
 
-    async def async_send_command(self, command, **kwargs):
-        """Send a command to a device."""
+    async def async_send_command(self, command: Iterable[str], **kwargs: Any) -> None:
+        """Send commands to a device."""
+        assert self.hass is not None
         await self.hass.async_add_executor_job(
             ft.partial(self.send_command, command, **kwargs)
         )
 
-    def learn_command(self, **kwargs):
+    def learn_command(self, **kwargs: Any) -> None:
         """Learn a command from a device."""
         raise NotImplementedError()
 
-    async def async_learn_command(self, **kwargs):
+    async def async_learn_command(self, **kwargs: Any) -> None:
         """Learn a command from a device."""
+        assert self.hass is not None
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5012212)